demo/public/javascripts/demo.js

```diff
@@ -0,0 +1,87 @@
+(function () {
+    let otherLoads;
+
+    if (window.onload) {
+        otherLoads = window.onload;
+    }
+
+    let updateBackground = function () {
+        const el = document.getElementById("tsparticles");
+        const options = tsParticles.domItem(0).options;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['tsParticles' is not defined.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5136254)